volume-group-operator/controllers/utils/volumegroup.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"github.com/IBM/volume-group-operator/api/v1"
+	volumegroupv1 "github.com/IBM/volume-group-operator/api/v1"
```

Comment:
and we need to decide how do we want to call our API from the code (can be in a ticket), because we call it [`volumegroupv1`](https://github.com/IBM/ibm-block-csi-operator/blob/feature/CSI-4969_volumegroup_controller_implementation/volume-group-operator/controllers/utils/volumegroup.go#L5) and [`csiv1`](https://github.com/IBM/ibm-block-csi-operator/blob/feature/CSI-4969_volumegroup_controller_implementation/volume-group-operator/main.go#L38)

Reply:
done